src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export function getLinkDataCy(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return 'prevLink';
+    case totalPages + 1:
+      return 'nextLink';
+    default:
+      return 'pageLink';
+  }
+}
+
+export function getLinkHref(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return '#prev';
+    case totalPages + 1:
+      return '#next';
+    default:
+      return `#${index}`;
+  }
+}
+
+export function getLinkTextContent(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return '«';
+    case totalPages + 1:
+      return '»';
+    default:
+      return `${index}`;
+  }
+}
+
+export function handleClickPagination(
+  event: React.MouseEvent<HTMLUListElement>,
+  currentPage: number,
+  pagesAmount: number,
+  onPageChange: (page: number) => void,
+): void {
```

Comment:
The `onPageChange` function type can be simplified to '(page: number) => void' without the comma at the end, as the last parameter in the function signature does not require a trailing comma.